packages/clay-css/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "clay-css",
-  "version": "2.10.0",
+  "name": "@clayui/css",
+  "version": "3.0.0",
```

Comment:
Should this also be a milestone or alpha?

Reply:
The version change of the packages will be made later in the release, when running a `lerna publish`, I'm not altering now. it will change too.